packages/create-next-app/templates/default/next.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  eslint: true,
+}
```

Comment:
@timneutkens I'm not sure if we're in favor of having a default `next.config.js` in Create Next App templates, but I was hesitant to make this an opt-out flag. Could be jarring for so many Next.js users who are using ESLint only through their editor to see this enabled at build time automatically.

Happy to consider others alternative however 👍